### DISCUSSION TITLE
Shoonya v1.1

### DISCUSSION THREAD

**Discussion by aravinth** [AI4Bharat/Shoonya#11] [Announcements]
This release focuses on a few bug fixes, and covers some user requests, along with dataset automation functions mentioned in detail below:
- Rename Task status `rejected` to `to_be_revised`.
- Fix a bug in the review feature.
- Add reviewer reports for project analytics.
- Refactor `User` field to `Annotator` in Project model.
- Functionality to remove users from workspace.
- Code refactoring to take `user_id` instead of `username` or `email` in various endpoints.
- Support automated transformation of Sentence Text Datasets to Translation Pairs Dataset involving functions for Generating Machine Translations using IndicTrans and Google Translate models.

<hr /><em>This discussion was created from the release <a href='https://github.com/AI4Bharat/Shoonya/releases/tag/v1.1'>Shoonya v1.1</a>.</em>

**Comment by prashantkodali** [comment]
Hello Aravinth, 
Can this version of Shoonya be hosted locally? If yes, could you point me to the relevant how-to page.

**Reply by aravinth** [reply]
Hi Prashant,

Yes, you should be able to set it up in your local.
For Backend; https://www.youtube.com/watch?v=uGuUnsPqj4c
For Frontend; https://github.com/AI4Bharat/Shoonya-Frontend

Please let us know in case the steps requires any refinement.
